src/analysis/AnalysisClient.js

```diff
@@ -13,14 +13,18 @@ class AnalysisClient {
       approxTaxRate - accounts for tax on item when purchasing from walmart
       effectiveValueOfOurDollar - because of sites like giftcardgranny.com and eBay, for every dollar we spend
       at walmart, we could actually be spending less e.g. 98 cents.
-      ROIThreshold - the cutoff ROI for items we want to consider buying
+      ROIThreshold - the cutoff ROI for items the item being considered e.g. 25
+      minRatingsValue - minimum average rating of the item being considered e.g. 3.0
+      minReviewsCount - minimum number of reviews of the item being considered e.g. 30
   */
-  constructor() {
+  constructor(analysisParameters) {
     this.fbaShippingCostPerPound = 0.50; 
     this.approxTaxRate = 0.06;
     this.analyzedProductsInfo = [];
     this.effectiveValueOfDollar = 0.98;
-    this.ROIThreshold = 25;
+    this.ROIThreshold = parseInt(analysisParameters.minROI);
```

Comment:
This class should be receiving the parameters in the correct format. Even your documentation above gives an integer as an example. You should parse the strings to ints as soon as they are received from the command line.